BillingApp/BillingApp/settings.py

```diff
@@ -46,7 +46,7 @@
 TEMPLATES = [
     {
         'BACKEND': 'django.template.backends.django.DjangoTemplates',
-        'DIRS': [],
+        'DIRS': [os.path.join(BASE_DIR, 'templates')],
```

Comment:
why we need to add this?
Actually we could access `app/templates` if APP_DIRS is True